lisa/features/security_profile.py

```diff
@@ -55,4 +95,9 @@ def enabled(self) -> bool:
         return True
 
 
-SecureBootEnabled = partial(SecurityProfileSettings, secure_boot_enabled=True)
+SecureBootEnabled = partial(
+    SecurityProfileSettings,
+    security_profile=search_space.SetSpace(
+        True, [SecurityProfileType.Boot, SecurityProfileType.CVM]
```

Comment:
Leave only Boot here, because it's used as the test case requirement, not capability.

Reply:
Here is the case I was considering when I wrote this:

Test case requires SecureBoot
Runbook specifies CVM

Is it okay to run the test case with CVM security profile?

Reply:
Yes, it sounds the test case should be run, and it needs to make the check method correct.